src/main/java/seedu/address/logic/parser/DeleteCommandParser.java

```diff
@@ -29,7 +30,7 @@ public DeleteCommand parse(String args) throws ParseException {
         } else if (flag == FLAG_INTERVIEW) {
             return new DeleteInterviewCommandParser().parse(argsWithoutFlag);
         } else if (flag == FLAG_POSITION) {
-            // calls delete position command parser
+            return new DeletePositionCommandParser().parse(argsWithoutFlag);
         }
 
         return null;
```

Comment:
Maybe there is a better way of handling this? Since return null might give a `nullpointerexception` if it ever reaches this portion of the code. This could theoretically happen if we change how `ArgumentTokenizer.getFlag` works.